ui/raidboss/data/07-dt/alliance/cloud_of_darkness_chaotic.ts

```diff
@@ -0,0 +1,414 @@
+  'tankCleave': '0156',
+  // flare marker
+  // Offsets: 76147,157739
+  // Vfx Path: all_at8s_0v
+  'flareMarker': '015A',
+  // player who will drop a bramble
+  // Vfx Path: m0302_seed_8s_x
+  'brambleSeed': '0227',
+  // countdown to hand spawning on player
+  // Offsets: 86301,97364,104403,111399,145339,153274,165355
+  // Vfx Path: l1rz_grasp_count5s_0x
+  'handCountdown': '0228',
+  // clockwise rotation
+  // Vfx Path: turnright_14m_14s_c0x
+  'rotateClockwise': '0234',
+  // counter-clockwise rotation
+  // Vfx Path: turnleft_14m_14s_c0x
+  'rotateCounterClockwise': '0235',
+  // bramble about to tether to a player
+  // Vfx Path: exclamation_8s_x
+  '0239': '0239',
+} as const;
+
+const triggerSet: TriggerSet<Data> = {
+  id: 'TheCloudOfDarknessChaotic',
+  zoneId: ZoneId.TheCloudOfDarknessChaotic,
+  timelineFile: 'cloud_of_darkness_chaotic.txt',
+  initData: () => ({
+    deadlyEmbraceDodgeDirectionCollected: 'unknown',
+    deadlyEmbraceDodgeDirection: 'unknown',
+  }),
+  timelineTriggers: [],
+  triggers: [
+    {
+      id: 'Cloud Chaotic Inner/Outer Darkness Gain',
+      type: 'GainsEffect',
+      netRegex: { effectId: ['1051', '1052'], capture: true },
+      condition: Conditions.targetIsYou(),
+      run: (data, matches) => {
+        switch (matches.effectId) {
+          case '1051':
+            data.innerDarkness = true;
+            break;
+          case '1052':
+            data.outerDarkness = true;
+            break;
+        }
+      },
+    },
+    {
+      id: 'Cloud Chaotic Inner/Outer Darkness Lose',
+      type: 'LosesEffect',
+      netRegex: { effectId: ['1051', '1052'], capture: true },
+      condition: Conditions.targetIsYou(),
+      run: (data, matches) => {
+        switch (matches.effectId) {
+          case '1051':
+            delete data.innerDarkness;
+            break;
+          case '1052':
+            delete data.outerDarkness;
+            break;
+        }
+      },
+    },
+    {
+      id: 'Cloud Chaotic Doom',
+      type: 'GainsEffect',
+      netRegex: { effectId: 'D24', capture: true },
+      condition: (data, matches) => {
+        // this can get very noisy if we trigger on every player that receives Doom,
+        // so limit to only calling for players in your immediate party.
+        return data.CanCleanse() && data.party.inParty(matches.target);
+      },
+      alertText: (data, matches, output) => {
+        return output.text!({ player: data.party.member(matches.target) });
+      },
+      outputStrings: {
+        text: {
+          en: 'Cleanse ${player}',
+          de: 'Reinige ${player}',
+          fr: 'Guérissez ${player}',
+          cn: '康复 ${player}',
+          ko: '${player} 디버프 해제',
+        },
+      },
+    },
+    {
+      id: 'Cloud Chaotic Blade of Darkness',
+      type: 'StartsUsing',
+      netRegex: { id: ['9DFD', '9DFB', '9DFF'], source: 'Cloud of Darkness', capture: true },
+      suppressSeconds: 2,
+      infoText: (data, matches, output) => {
```

Comment:
Add a conditional duration for the followup. 7 seconds felt okay with my extremely quick testing but 10 seconds might make more sense. IIRC it's 6.7s for the `Blade of Darkness` cast plus 5.3 seconds for the followup mechanic, but they overlap some.

```suggestion
      netRegex: { id: ['9DFD', '9DFB', '9DFF'], source: 'Cloud of Darkness', capture: true },
      durationSeconds: (data) => data.bladeOfDarknessFollowup !== undefined ? 7 : 3,
      suppressSeconds: 2,
      infoText: (data, matches, output) => {
```